src/components/PageHeading.js

```diff
@@ -1,22 +1,75 @@
 import { Component } from 'react';
 import { div, hh, img, h2, span } from 'react-hyperscript-helpers';
-import './PageHeading.css';
 
 export const PageHeading = hh(class PageHeading extends Component {
 
+  margins(iconSize) {
+
+    const MEDIUM = {
+      marginLeft: '55px'
+    };
+
+    const LARGE = {
+      marginLeft: '70px'
+    };
+
+    const NONE = {
+      marginLeft: '0'
+    };
+
+    if (iconSize === 'none') {
+      return NONE;
+      }
+      else {
```

Comment:
nit: can simplify this to multiple `if (case1) {} if (case2) {}` before the final `else {}`